src/extensions/Array.test.ts

```diff
@@ -0,0 +1,133 @@
+import { describe } from 'riteway'
+
+import './Array'
+
+describe('Array.prototype.filterTruthy', async (should: any) => {
+  const { assert } = should()
+
+  {
+    const arr = ['truthy', true, 'works', 5]
+
+    assert({
+      given: 'array of truthy values',
+      should: 'return the full array',
+      actual: arr.filterTruthy(),
+      expected: ['truthy', true, 'works', 5]
+    })
+  }
+
+  {
+    const arr = ['truthy', true, false, undefined, null]
+
+    assert({
+      given: 'array of truthy and falsy values',
+      should: 'return the truthy values',
+      actual: arr.filterTruthy(),
+      expected: ['truthy', true]
+    })
+  }
+
+  {
+    const arr = [false, undefined, null]
+
+    assert({
+      given: 'array of falsy values',
+      should: 'return an empty array',
+      actual: arr.filterTruthy(),
+      expected: []
+    })
+  }
+})
+
+describe('Array.prototype.toObject', async (should: any) => {
+  const { assert } = should()
+
+  {
+    const arr = [['key1', 'value1'], ['key2', 'value2']]
+    const cb = (arr: ReadonlyArray<any>) => ({
+      key: arr[0],
+      value: arr[1]
+    })
+    assert({
```

Comment:
Can we add an empty line here? It will keep the formatting consistent with the rest of the file.